semantic_conventions/http-common.yaml

```
@@ -9,6 +9,16 @@ groups:
         requirement_level: required
         brief: 'HTTP request method.'
         examples: ["GET", "POST", "HEAD"]
+        note: |
+          The HTTP request method usually contains one of a standard methods defined in [RFC9110](https://www.rfc-editor.org/rfc/rfc9110.html#name-methods)
+          or [RFC5789](https://www.rfc-editor.org/rfc/rfc5789.html).
+
+          Instrumentations SHOULD call into TODO advice API (when available) before recording any telemetry
+          and specify a list of known HTTP request methods. For general-purpose HTTP instrumentations this list SHOULD include common request methods:
+          `GET`, `HEAD`, `POST`, `PUT`, `PATCH`, `DELETE`, `CONNECT`, `OPTIONS`, `TRACE`.
+          Applications MAY use TODO advice API to override instrumentation defaults.
```

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view

```suggestion
          Applications MAY use TODO View API to override instrumentation defaults.
```